graal/src/main/java/io/micronaut/graal/reflect/GraalTypeElementVisitor.java

```diff
@@ -261,16 +259,6 @@ private void generateNativeImageProperties(VisitorContext visitorContext) {
         try {
             String path = buildNativeImagePath(visitorContext);
             String reflectFile = path + REFLECTION_CONFIG_JSON;
-            String propsFile = path + NATIVE_IMAGE_PROPERTIES;
-
-            visitorContext.visitMetaInfFile(propsFile).ifPresent(gf -> {
-                visitorContext.info("Writing " + NATIVE_IMAGE_PROPERTIES + " file to destination: " + gf.getName());
-                try (PrintWriter w = new PrintWriter(gf.openWriter())) {
-                    w.println("Args = -H:ReflectionConfigurationResources=${.}/reflection-config.json");
-                } catch (IOException e) {
-                    visitorContext.fail("Error writing " + NATIVE_IMAGE_PROPERTIES + ": " + e.getMessage(), null);
-                }
-            });
```

Comment:
Why do you delete this?

Reply:
if the file is named `reflect-config.json` it is automatically picked up. We were only generating this file to load our inconsistently named `reflection-config.json` file

Reply:
You are right. Then I think you can rename `REFLECTION_CONFIG_JSON` to `REFLECT_CONFIG_JSON` here https://github.com/micronaut-projects/micronaut-core/pull/4541/files#diff-a6f6eed2d556fc4571d32b10ec2a16d17cb98075c9e1b3ae0395176281cbb061R73

For the rest of the changes, everything looks good.

Reply:
That is already done